code/website/index.html

```diff
@@ -3,11 +3,18 @@ <h2>Drasil - Generate All the Things!</h2>
 <hr>
 <br>
 <p>
+  Drasil is a framework for generating all of the software artifacts from a stable knowledge base,
+  focusing currently on scientific software. The main goals are to reduce knowledge duplication and
+  improve traceability. The atifacts are generated from a common knowledge-base using recipes
+  written in a Domain-Specific Language (DSL). These recipes allow us to specify which pieces of
+  knowledge should be used in which artifacts, how to transform them, and more.
+  <br><br>
   This webpage is designed to contain the most up to date
-  <a href="#caseStudies">case study artifacts</a>
+  <a href="#caseStudies">case study artifacts</a>, <a href="#haskDoc">documentation</a>,
   and <a href="#packages">package dependency graphs</a> from the Drasil repository.
   The case study artifacts include the Software Requirements Specification (SRS) for the case study,
   which specifies what the program sets out to achieve.
+  The Haskell Documentation section contains the current documentation for the Drasil framework.
   The package dependency graphs shows the hierarchy of modules within each package.
   The footer of this page contains the continuous integration build of the project,
   as well as the commit number that the build and artifacts are based off of.
```

Comment:
The changes for documentation prompted this comment, but we should have done it before. Any section which is `if`d below (like documentation), should have the same guards in the section, so we don't generate a link to a non-existent documentation section when not present, and similarly don't describe it.

Reply:
So far, Documentation is the only `if` that is also used in the intro. 

Case study artifacts and dependency graphs use `for` so I can not make this change for them (unless I rebuild them using the new `maybeListFieldWith`.

Reply:
Adding a conditional for the preamble about the docs should be sufficient.